inference-engine/src/cldnn_engine/cldnn_config.cpp

```diff
@@ -221,6 +222,19 @@ void Config::UpdateFromMap(const std::map<std::string, std::string>& configMap)
             } else {
                 THROW_IE_EXCEPTION_WITH_STATUS(NotFound) << "Unsupported KEY_CLDNN_ENABLE_FP16_FOR_QUANTIZED_MODELS flag value: " << val;
             }
+        } else if (key.compare(CLDNNConfigParams::KEY_CLDNN_MAX_NUM_THREADS) == 0) {
+            int max_threads = std::thread::hardware_concurrency();
```

Comment:
https://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency
It can return 0 if the environment does not defined the available concurrency. 
I think we need to set it as 1 if it returns 0.